Mage.Sets/src/mage/cards/d/DampingSphere.java

```diff
@@ -0,0 +1,148 @@
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.cards.d;
+
+import java.util.UUID;
+import mage.MageObject;
+import mage.Mana;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.ReplacementEffectImpl;
+import mage.abilities.effects.common.cost.SpellsCostIncreasementAllEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.events.GameEvent.EventType;
+import mage.game.events.ManaEvent;
+import mage.util.CardUtil;
+import mage.watchers.common.CastSpellLastTurnWatcher;
+
+/**
+ *
+ * @author L_J
+ */
+public class DampingSphere extends CardImpl {
+
+    public DampingSphere(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{2}");
+
+        // If a land is tapped for two or more mana, it produces {C} instead of any other type and amount.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new DampingSphereReplacementEffect()));
+
+        // Each spell a player casts costs {1} more to cast for each other spell that player has cast this turn.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new DampingSphereIncreasementAllEffect()), new CastSpellLastTurnWatcher());
+    }
+
+    public DampingSphere(final DampingSphere card) {
+        super(card);
+    }
+
+    @Override
+    public DampingSphere copy() {
+        return new DampingSphere(this);
+    }
+}
+
+class DampingSphereReplacementEffect extends ReplacementEffectImpl {
+
+    DampingSphereReplacementEffect() {
+        super(Duration.WhileOnBattlefield, Outcome.Neutral);
+        staticText = "If a land is tapped for two or more mana, it produces {C} instead of any other type and amount";
+    }
+
+    DampingSphereReplacementEffect(final DampingSphereReplacementEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public DampingSphereReplacementEffect copy() {
+        return new DampingSphereReplacementEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        return true;
+    }
+
+    @Override
+    public boolean replaceEvent(GameEvent event, Ability source, Game game) {
+        ManaEvent manaEvent = (ManaEvent) event;
+        Mana mana = manaEvent.getMana();
+        mana.setToMana(Mana.ColorlessMana(1));
+        return false;
+    }
+
+    @Override
+    public boolean checksEventType(GameEvent event, Game game) {
+        if (event.getType() == EventType.TAPPED_FOR_MANA) {
+            ManaEvent manaEvent = (ManaEvent) event;
+            Mana mana = manaEvent.getMana();
+            return mana.count() > 1;
+        }
```

Comment:
Is there are reason you're doing the applies logic in the check event type method?

Reply:
Not really, it should fit in either. Does it make a difference in this case?

Reply:
Not that I am aware of at this time. Just seems like something SonnarQube would call a code smell is all.

Reply:
I added the checksEventType method some time ago to get a fast precheck to exclude continous effects for further checking during the ApplyEffects method. 

So it makes sense at least for some uniformity to only check the event type there, although the functional outcome is the same.

Reply:
Switched mana.count() checking into applies method.